packages/node_modules/nav-frontend-stegindikator/src/stegindikator.tsx

```diff
@@ -194,17 +194,23 @@ class NavFrontendStegindikator extends React.Component<NavFrontendStegindikatorP
                     ref={(list: HTMLOListElement) => { this.list = list; }}
                 >
                     {
-                        this.props.steg.map((steg, i) => (
-                            <StegindikatorSteg
-                                index={(i + 1)}
-                                label={steg.label}
-                                visLabel={this.state.visLabel}
-                                key={`${steg.label.split(' ').join('')}`}
-                                aktiv={this.props.aktivtSteg === (i + 1)}
-                                ferdig={this.props.ferdigSteg && this.props.ferdigSteg.indexOf(i + 1) !== -1}
-                                {...steg}
-                            />
-                        ))
+                        this.props.steg.map((steg, i) => {
```

Comment:
Et siste spørsmål her fra meg @Lillebo - trenger vi egentlig disse i+1 tingene bortsett fra når man skal legge på tallet på labelen i selve `StegindikatorSteg`-komponenten? Mye `(i+1)` i koden gjør koden litt vanskelig å lese, og at index+i settes på en `index` prop tenker jeg også er litt uheldig når det ikke faktisk er den opprinnelige index-verdien, men index+1. Tanker om det?

Reply:
Nja, var litt i tvil om det var mest brukervennlig å legge opp til zero-based indexer eller ikke. Følte kanskje at ikke-zero-based virket litt mer naturlig da vi samtidig rendrer stegene med index + 1. Men kan godt endre dette hvis dere ønsker :)

Reply:
Skjønner. Da tenker ihvertfall jeg at det er bedre å legge inn den +1-kompleksiteten isolert til labelen i `StegindikatorSteg`-komponenten, enn at vi må gjøre justeringer og legge på mer kompleksitet i `Stegindikator` for at det skal funke riktig i `StegindikatorSteg`.

Reply:
Hm, bør vi ikke gjøre dette i `Stegindikator` hvis den fortsatt skal ha ikke-zero-based input for `aktivtSteg` og `ferdigSteg`? Kan uansett sikkert erstatte alle forekomstene av `i + 1` med en `index`-variabel som opprettes sammen med `iFerdigSteg`, `preAktivtSteg` og `ferdig` for å gjøre det litt mer lesbart.

Reply:
Jeg tenker at `Stegindikator` bør basere seg på å starte på 0 for å angi aktivtSteg og ferdigeSteg, om det er sånn at man sender inn en array med steg. Tenker at jeg ville blitt overrasket om aktivtSteg da må settes til 1 for å gjøre steget på index 0 aktivt (osv).

Reply:
OK, skal fikse 👍